hack/lib/golang.sh

```diff
@@ -306,8 +306,8 @@ readonly KUBE_TEST_SERVER_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]:+"${KUBE_SERVER
 
 # Gigabytes necessary for parallel platform builds.
 # As of January 2018, RAM usage is exceeding 30G
-# Setting to 40 to provide some headroom
-readonly KUBE_PARALLEL_BUILD_MEMORY=40
+# Defaulting to 40G to provide some headroom
```

Comment:
This comment doesn't make sense anymore if this isn't hard coded

Reply:
Added a note that it can be overwritten at our own risk, whereas it still defaults to 40GB.